.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -708,12 +708,9 @@ Using `tsh` join an SSH session as two moderators (two separate terminals, role
 
 ## Performance
 
+### Scaling Test
```

Comment:
IIRC based on recent discussion we were considering rewriting this section to be something like this:

```
Scale to 10k reverse tunnel nodes in the following configurations:
 - [ ] etcd
 - [ ] DynamoDB
 - [ ] Firestore
 - [ ] Cloud
 
 Perform the following additional scaling tests on DynamoDB:
- [ ] 10k direct dial nodes.
- [ ] 500 trusted clusters.
```

Reply:
Reasoning for using 10k tunnel nodes as our only "all backends" scaling test is that it induces strictly more load (and of the same kinds) as the other scaling scenarios, making them redundant in terms of testing backend-specific issues.

I'd forgotten that we opted to keep cloud scaling more limited for now... updated suggestion:

```
Perform reverse tunnel node scaling tests for all backend configurations:
 - [ ] etcd - 10k (30k in the future)
 - [ ] DynamoDB - 10k
 - [ ] Firestore - 10k
 - [ ] Cloud - 1k
 
 Perform the following additional scaling tests on DynamoDB:
- [ ] 10k direct dial nodes.
- [ ] 500 trusted clusters.
```